src/youtube.js

```diff
@@ -332,6 +332,12 @@ class Youtube extends FakeEventTarget implements IEngine {
    */
   seekToLiveEdge(): void {}
 
+  // eslint-disable-next-line no-unused-vars
+  attachMediaSource(playbackEnded: ?boolean): void {}
```

Comment:
can't just ignore the param?

Reply:
I didn't try but I think it better to keep the signature right

Reply:
no. better to avoid using a particular eslint rule